vulnerability-analyzer/src/main/java/org/hyades/config/InternalScannerConfig.java

```diff
@@ -9,4 +10,6 @@ public interface InternalScannerConfig {
 
     int topicPartitions();
 
+    @WithDefault("false")
+    boolean aliasSyncEnabled();
```

Comment:
The internal analyzer doesn't have any data that the API server doesn't already know, and it's only returning source and vulnId in its result. I think we can drop this config property.